tests/test_views.py

```diff
@@ -2903,8 +2884,9 @@ def test_pointer_list_read_contributor_cannot_remove_private_component_entry(sel
     def test_pointer_list_read_contributor_cannot_remove_public_component_entry(self):
         url = web_url_for('view_project', pid=self.project._id)
 
-        self.project.add_pointer(ProjectFactory(creator=self.user),
-                                 auth=Auth(user=self.user))
+        linked_to = ProjectFactory(creator=self.user)
+        linked_to.update(fields={'is_public':1})
```

Comment:
You should be able to specify arbitrary fields when using factories, as though you were instantiating an object of that type -- these three lines would be equivalent to:

```python
self.project.add_pointer(ProjectFactory(creator=self.user, is_public=True),
                         auth=Auth(user=self.user))
```

That said, this isn't a blocker